attributes.rst

```diff
@@ -0,0 +1,146 @@
+.. index::
```

Comment:
We should move this document under `reference/attributes.rst` and mention it in `reference/map.rst.inc`